umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/iso_shared/distributions_related_url.clj

```diff
@@ -84,12 +91,66 @@
            {:codeList (str (:ngdc iso/code-lists) "#CI_OnLineFunctionCode")
             :codeListValue code}]]]])))
 
+(defn generate-online-resource-url2
```

Comment:
Is there also a `generate-online-resource-url` function? If so it's probably worth mentioning in this docstring why there are two and how to choose which one to use.